### DISCUSSION TITLE
structure_cond_seq

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/labview_book#62] [Announcements]
# 条件结构和顺序结构

https://lv.qizhen.xyz/structure_cond_seq

**Comment by XuY6969** [comment]
有没有答案呀

**Reply by zsl-nuaa** [reply]
我做的 仅供参考
https://github.com/zsl-nuaa/labview-practise/blob/main/test1.png

**Comment by 517LE** [comment]
“避免把控件放在条件结构内”的下一行，“一个数实数数组“input array””这句话里的“是”打成"数"了吧？

**Reply by ruanqizhen** [reply]
已修改，多谢指正！

**Comment by SJH0912** [comment]
大佬，练习题目答案有吗

**Reply by ruanqizhen** [reply]
没有答案。自己编写一下，看到程序运行结果就表示成功了。

**Comment by Automan-wfq** [comment]
在“是否要设置默认分支？”环节中，少写了一个字，原文是“被客户现它的结果不对，那问题就严重了。”，应该是“被客户发现它的结果不对，那问题就严重了。”。

**Reply by ruanqizhen** [reply]
已修改，多谢指正！
